pkg/install/install.go

```diff
@@ -487,3 +490,13 @@ func (i *Installer) deployARMTemplate(ctx context.Context, rg string, tName stri
 
 	return err
 }
+
+// addResourceProviderVersion sets the deploying resource provider version in the cluster document for deployment-tracking purposes.
+func (i *Installer) addResourceProviderVersion(ctx context.Context) error {
+	var err error
+	i.doc, err = i.db.PatchWithLease(ctx, i.doc.Key, func(doc *api.OpenShiftClusterDocument) error {
+		i.doc.OpenShiftCluster.Properties.ProvisionedBy = version.GitCommit
```

Comment:
```suggestion
		doc.OpenShiftCluster.Properties.ProvisionedBy = version.GitCommit
```

^ that's an important one.